plugins/plugin-chart-table/src/Styles.tsx

```diff
@@ -21,6 +25,10 @@ export default styled.div`
   th.is-sorted svg {
     color: #a8a8a8;
   }
+  .table > tbody > tr:first-of-type > td,
+  .table > tbody > tr:first-of-type > th {
+    border-top: 0;
+  }
```

Comment:
This is needed since previously there is `border-collapse` between `thead` and `tbody`. When they are placed into different containers, the borders won't collapse anymore.